index.html

```diff
@@ -969,7 +662,12 @@
         <div class="header item">FFX|V Fish Tracker App</div>
         <div class="right item">
           <b>Current Time:</b>&nbsp;<span id="eorzeaClock">##:##</span>&nbsp;(<span id="earthClock">#########</span>)
-          <button id="theme-button" class="theme-button"></button>
+          <div class="theme-switch-wrapper">
```

Comment:
I would prefer this use the Semantic UI instead.
See https://semantic-ui.com/modules/checkbox.html#toggle

Reply:
done